packages/compose/src/hooks/use-async-list/index.ts

```diff
@@ -56,15 +56,13 @@ function useAsyncList< T >(
 		}
 		setCurrent( firstItems );
 
-		const asyncQueue = createQueue();
+		const asyncQueue = createQueue( { once: true } );
 		for ( let i = firstItems.length; i < list.length; i += step ) {
 			asyncQueue.add( {}, () => {
-				flushSync( () => {
```

Comment:
I think it could be fine to keep this regardless.

Reply:
Fine to keep but probably not needed?

Reply:
Otherwise, is there a point to do the config?

Reply:
> Otherwise, is there a point to do the config?

The idea is that maybe there's remaining time after the first preview rendering but not enough to render the next one, so the "once" force us the priority queue to schedule an update for later, give a decent amount of time for the browser for potential "interactions" that may come during the first rendering or a bit after but before the next "idle callback". The ultimate goal to reduce the lags as much as we can.

Reply:
Oh, ok, got it :)